src/test/java/com/twilio/sdk/resource/instance/taskrouter/WorkflowTest.java

```diff
@@ -95,6 +95,45 @@ public void testCreateWorkflowWithHelper() throws Exception {
 		assertNull(workflow.getFallbackAssignmentCallbackUrl());
 		assertEquals(120, workflow.getTaskReservationTimeout().intValue());
 	}
+	
+	@Test
+		public void testSerializeWorkflowFriendlyName() throws Exception {
```

Comment:
I split the test I initially had into two -- one on friendly_name and one on filter_friendly_name. I made some changes though to make it more readable.